test/controllers/PenaltyDetailsController.test.ts

```diff
@@ -71,6 +71,7 @@ describe('PenaltyDetailsController', () => {
                 .expect(response => {
                     expect(response.get('Set-Cookie')).to.contain('penalty-cookie=1; Path=/');
                     expect(response.status).to.be.equal(MOVED_TEMPORARILY);
+                    expect('Location', OTHER_REASON_DISCLAIMER_PAGE_URI);
```

Comment:
That is not complete assertion. It does not fail but at the same time it does not assert anything.